.github/workflows/setup-conda.yml

```diff
@@ -0,0 +1,31 @@
+# Verify that the setup-conda action works
+
+name: Validate setup-conda action
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  conda:
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest, windows-latest]
+        conda: [anaconda, miniconda]
+
+    runs-on: ${{ matrix.os }}
+    name: ${{ matrix.os }}-${{ matrix.conda }}
+
+    steps:  
+    - name: Setup conda
+      uses: iiasa/actions/setup-conda@main
```

Comment:
Given the error messages [here](https://github.com/iiasa/actions/runs/4153994166?check_suite_focus=true#step:1:31), I think we can change this line to:
```
uses: iiasa/action/setup-conda@feature/setup-conda-action
```

This will use the version on this same branch. Then we can change it back to `@main` after this PR is merged.